homeassistant/components/hassio/__init__.py

```diff
@@ -395,6 +396,16 @@ async def push_config(_: Event | None) -> None:
 
     async def async_service_handler(service: ServiceCall) -> None:
         """Handle service calls for Hass.io."""
+        if service.service == SERVICE_ADDON_UPDATE:
+            async_create_issue(
+                hass,
+                DOMAIN,
+                "update_service_deprecated",
+                breaks_in_ha_version="2025.5",
+                is_fixable=False,
```

Comment:
We normally make these issues fixable and persistent do the user can fix them when they've updated their automations and scripts as we should ask them to do.